src/validators/auth.ts

```diff
@@ -23,5 +23,12 @@ export function authValidator(method: string): ValidationChain[] {
                 body("password", "Invalid or missing 'password'").isString().exists(),
             ];
         }
+        case "POST /passwordReset": {
+            return [
+                body("oldPassword", "Invalid or missing 'oldPassword'").isString().exists(),
+                body("newPassword", "Invalid or missing 'newPassword'").isString().exists(),
+                body("jwtToken", "Invalid or missing 'jwtToken'").isString().exists()
```

Comment:
check for auth header after moving token out of body

Reply:
something like header("authorization", "invalid or missing token").exists() might work? not sure